### PR TITLE
Enable foreground tracking for everyone

### PR DESCRIPTION
**Pull request by ThibautGeriz**
## Motivation

Enable foreground tracking for everyone

## Changes

Remove experimental flag

---

I have gone over the [contributing](https://github.com/DataDog/browser-sdk/blob/main/CONTRIBUTING.md) documentation.

### CODE REVIEW

**Review by BenoitZugmeyer — APPROVED** [review]

File: packages/rum-core/src/domain/foregroundContexts.ts [hunk 1]
```diff
@@ -60,10 +51,6 @@ function addNewForegroundPeriod() {
   }
   const currentForegroundPeriod = foregroundPeriods[foregroundPeriods.length - 1]
   if (currentForegroundPeriod !== undefined && currentForegroundPeriod.end === undefined) {
-    addMonitoringMessage('Previous foreground periods not closed. Continuing current one', {
```
Comment (bcaudan) [inline packages/rum-core/src/domain/foregroundContexts.ts:63]: What about other messages?

Comment (ThibautGeriz) [reply packages/rum-core/src/domain/foregroundContexts.ts:63]: AFAIK, They are not triggered at the moment, I would like to know if we change something and those limit case start to trigger.

Comment (bcaudan) [reply packages/rum-core/src/domain/foregroundContexts.ts:63]: By looking at the logs I shared with you earlier, we can see some occurrences for all of them.

**Comment by ThibautGeriz** [comment]
Closed for https://github.com/DataDog/browser-sdk/pull/1053

**Pull request closed without merging**